sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/EnvironmentResourceTest.java

```diff
@@ -47,6 +48,36 @@ void resourceFromConfig() {
                 "MyApp"));
   }
 
+  @Test
+  void serviceName() {
+    Attributes attributes =
+        EnvironmentResource.getAttributes(
+            ConfigProperties.createForTest(
+                singletonMap(EnvironmentResource.SERVICE_NAME_PROPERTY, "myService")));
+
+    assertThat(attributes).isEqualTo(Attributes.of(ResourceAttributes.SERVICE_NAME, "myService"));
```

Comment:
I think you can use otel testing assertions if you want

Reply:
I'll take a look. I didn't remember there being attributes-related assertions.